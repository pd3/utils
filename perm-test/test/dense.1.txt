
perm-test -t test/dense.1.tgt.txt -b test/dense.1.bg.txt -f test/dense.1.fai -c test/dense.1.calls.txt -n 1e9

             10        20        30        40        50
    12345678901234567890123456789012345678901234567890123456
       bbb bbbb bbb     ttt   bbb   ttt ttt    bbbb tttt bbb
                    ccccccc
    ....... ....... ....ooo ....... ooo.ooo ....... xxxx...
     ....... ....... ...ooo. ....... oo.ooo. ....... xxx....
      ....... ....... ..ooo.. ......x ..ooo.. ......x
       ....... ....... .ooo... .....xx .ooo... .....xx
        ....... ....... xxx.... ....xx. ooo.... ....xxx
         ....... ....... xx..... ...ooo. xx..... ...xxxx
          ....... ......x x...... ..ooo.o x...... ..xxxx.
           ....... .....oo ....... .ooo.oo ....... .xxxx..

    exact: 30./50 = 0.6
    perm-test:      0.600010167
    err:   1.0167e-05

perm-test -t test/dense.1.tgt.txt -b test/dense.1.bg.txt -f test/dense.1.fai -c test/dense.1.calls.txt -n 1e9 --no-bg-overlap

    exact: 13./50 = 0.26
    perm-test:      0.259993827
    err:   6.173e-06


