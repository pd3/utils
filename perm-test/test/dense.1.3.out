# NITER_ROUNDS:
#    - number of iterations total
#    - number of batches
# TEST_ENR:
#    - number of iterations
#    - number of times the simulation had the same or more hits than observed in input data
#    - P-value derived from the two fields
# TEST_DPL:
#    - number of iterations
#    - number of times the simulation had the same or fewer hits than observed in input data
#    - P-value derived from the two fields
# TEST_FOLD:
#    - number of hits in the input data
#    - average number of hits in simulations
#    - average of standard deviations approximated from each batch using the current average estimate
# DIST:
#    - number of hits in a simulation
#    - number of simulations with this many hits
SEED	1577481419
NITER_ROUNDS	1.000000e+05	1
MSG	Maximum chromosome randomization error due to length discretization: 0.0e+00%
TEST_ENR	100000	25959	2.595900e-01
TEST_DPL	100000	100000	1.000000e+00
TEST_FOLD	1	0.259590	0.438410
DIST	0	74041
DIST	1	25959
